### DISCUSSION TITLE
在线配置

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#1657] [功能建议 / Ideas]
想搞个在线配置，类似于DuckDuckgo的那种，用口令储存。

**Comment by the1812** [comment]
![image](https://user-images.githubusercontent.com/26504152/111565260-04e80e00-87d6-11eb-9aae-9d414aebf2cc.png)

**Reply by FoundTheWOUT** [reply]
😃可以

**Reply by FoundTheWOUT** [reply]
看了一下，其实还是不太一样的吧，这玩意还是要手动上传和手动下载的，并不是真正意义的口令同步
